addon/templates/components/frost-error.hbs

```diff
@@ -0,0 +1,23 @@
+<div class="frost-error-description" data-test={{hook (concat hook '-description')}}>
+  {{description}}
+</div>
+<div class="frost-error-suggestion" data-test={{hook (concat hook '-suggestion')}}>
+  {{suggestion}}
+</div>
+<div class="frost-error-http-error" data-test={{hook (concat hook '-http-error')}}>
+  Error {{errorCode}} - {{errorTitle}}
+</div>
+<div class="frost-error-message" data-test={{hook (concat hook '-message')}}>
+  {{errorMessage}}
+</div>
+{{#frost-expand
+  class="frost-error-expand"
+  hook=(concat hook '-expand')
+  expanded=false
```

Comment:
This should be bound to the state property for expansion in the `frost-error` component js